sui/src/rpc_gateway.rs

```diff
@@ -120,10 +120,17 @@ impl RpcGatewayServer for RpcGatewayImpl {
         Ok(TransactionBytes::from_data(data)?)
     }
 
-    async fn get_owned_objects(&self, owner: SuiAddress) -> RpcResult<ObjectResponse> {
-        debug!("get_objects : {}", owner);
-        let objects = self.gateway.get_owned_objects(owner).await?;
-        Ok(ObjectResponse { objects })
+    async fn get_objects_owned_by_address(
+        &self,
+        address: SuiAddress,
+    ) -> RpcResult<Vec<ObjectInfo>> {
```

Comment:
The return type `ObjectInfo` is a bit confusing with the existing `get_object_info` method. Can we either rename `ObjectInfo` to `ObjectSummary` or rename the `get_object_info` to `get_object`?